dnf.spec

```diff
@@ -314,6 +314,9 @@ popd
 %systemd_post dnf-automatic.timer dnf-automatic-notifyonly.timer dnf-automatic-download.timer dnf-automatic-install.timer
 
 %preun automatic
+if [ ! -e %{_unitdir}/dnf5-automatic.timer ]; then
+    %systemd_preun dnf-automatic.timer
+fi
 %systemd_preun dnf-automatic.timer dnf-automatic-notifyonly.timer dnf-automatic-download.timer dnf-automatic-install.timer
```

Comment:
Shouldn't be `dnf-automatic.timer` removed from the list?

Reply:
You are right.

Reply:
I resolved it in [ed9a08f](https://github.com/rpm-software-management/dnf/pull/2234/commits/ed9a08fbef85108ca43303cb6886042910620c08)